.tool-versions

```diff
@@ -1,2 +1,2 @@
-erlang 23.2.6
-elixir 1.11.3
+erlang erlang 27.0.1
```

Comment:
Is it a typo?
```suggestion
erlang 27.0.1
```

Reply:
Good catch! Looks like I mistakenly ran `asdf set erlang erlang 27.0.1` which either just didn't work and `asdf` defaulted to my global Erlang version, or was somehow luckily still correctly parsed. Fixed.